autoload/neomake/makers/ft/java.vim

```diff
@@ -348,5 +365,26 @@ function! s:GetGradleClasspath() abort
     return ''
 endf
 
+function! s:GetAntClasspath() abort
+    let ant = s:findFileInParent('build.xml', expand('%:p:h', 1))
+    if s:has_ant && filereadable(ant)
+        if !has_key(g:neomake_java_javac_ant_ftime, ant) || g:neomake_java_javac_ant_ftime[ant] != getftime(ant)
+            try
+                let f = tempname()
```

Comment:
Unused var `f`.

Reply:
Argh, forgot to remove the `f` var after not using it. Neomake doesn't have a VimL lint? :-)

Reply:
Well, there is vint (which is fas, but does not detect thist), and vimlint (which is slow and therefore not enabled by default).
You also have to install some wrapper script for vimlint IIRC.

Reply:
We run it on Travis though, so that's fine in general.